Examples/Tests/reduced_diags_tools/analysis.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
+
+#####################################################
+# examples of usage of the reduced diagnostic tools #
+#####################################################
+
+import sys
+
+sys.path.append('../../../Python/pywarpx/diagnostics/data/')
+mydir = '.'
+
+##################
+# ParticleEnergy #
+##################
+
+import ParticleEnergy as PE
+
+diag = PE.ParticleEnergyData(fname='PartEne.txt')
+
+# extract some general info
+times = diag.get_times()
+print('available times\n {}'.format(times))
+
+steps = diag.get_steps()
```

Comment:
Could some `asserts` be added to check that the values obtained from the data files are correct?